Wrappers/Python/cil/optimisation/operators/Operator.py

```diff
@@ -217,28 +218,36 @@ def PowerMethod(operator, max_iteration=10, initial=None, tolerance = 1e-5,  ret
         i = 0
         while (i < max_iteration and diff > tolerance):
             i+=1

```

Comment:
we probably want to move the increment to the end of the loop as we are not running the iteration `i=0`

Reply:
Done!